app/models/icon_library.rb

```diff
@@ -0,0 +1,22 @@
+class IconLibrary

```

Comment:
Oooh I like this!

Reply:
Thanks! It worked even better than I thought too! I was able to move the image selecting into a method in Piece.rb itself, and it should be super easy to change the images now if we'd like!